content/4_Gold/Divis.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: divis
 title: 'Divisibility'
-author: Darren Yao, Michael Cao, Andi Qu, and Kevin Sheng
+author: Darren Yao, Michael Cao, Andi Qu, and Kevin Sheng, Jesse Choe
```

Comment:
why? you didn't add anything to the module...

Reply:
Two solutions lol (is that really nothing?)

Reply:
the solutions are internal sols, so you're credited there... internal solution authors aren't added to the module

Reply:
^

Reply:
ok cool u guys can remove then.